tests/runners/airflow/tasks/test_python.py

```diff
@@ -59,14 +76,14 @@ def test_apply_task_to_dag(self):
 
         for task in dag.tasks:
             print(f'Executing task {task.task_id}')
-            task.execute({})
+            task.execute(DummyDag('my_dag', task.task_id).context)
```

Comment:
why is this needed?

Reply:
It was missing. It didn't have a DAG definition to test against it.
@zionrubin

Reply:
if the tests were failing, need to take a look. 
if not -> perhaps separate from this commit